mock/datasets/no2.data.mdx

```diff
@@ -46,6 +46,8 @@ layers:
           return `${dateFns.format(datetime, 'LLL yyyy')} VS ${dateFns.format(compareDatetime, 'LLL yyyy')}`;
         }
     legend:
+      unit: 
+        label: Molecules cm3
       type: gradient
       min: "Less"
       max: "More"
```

Comment:
Unit defined as part of the legend. Following the structure suggested in https://github.com/NASA-IMPACT/veda-ui/issues/356#issuecomment-1329378711